### PR TITLE
Fix visiblity state of model map layers

### PR DESCRIPTION
**Pull request by mdastous-bentley**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

File: core/frontend/src/tile/map/MapTileTree.ts [hunk 1]
```diff
@@ -1001,9 +1001,11 @@ export class MapTileTreeReference extends TileTreeReference {
 
     for (; treeIndex < this._layerTrees.length; treeIndex++) {
       const layerTreeRef = this._layerTrees[treeIndex];
+      const isImageryMapLayer = layerTreeRef instanceof ImageryMapLayerTreeReference;
       // Load tile tree for each configured layer.
-      // Note: Non-visible layer are also added to allow proper tile tree scale range visibility reporting.
+      // Note: Non-visible imagery layer are always added to allow proper tile tree scale range visibility reporting.
       if (layerTreeRef && TileTreeLoadStatus.NotFound !== layerTreeRef.treeOwner.loadStatus
```
Comment (ben-polinsky) [inline core/frontend/src/tile/map/MapTileTree.ts:1007]: Suggestion: This conditional is getting a bit unwieldy. Maybe extract it to a function or extract each of the sub-conditions into its own variable?

Comment (mdastous-bentley) [reply core/frontend/src/tile/map/MapTileTree.ts]: should be better now

Comment (mdastous-bentley) [reply core/frontend/src/tile/map/MapTileTree.ts]: @ben-polinsky

**Comment by mdastous-bentley** [comment]
@Mergifyio backport release/4.5.x

**Pull request merged**